thrust/thrust/pair.h

```diff
@@ -30,6 +30,8 @@
 #  pragma system_header
 #endif // no system header
 
+#include <thrust/type_traits/is_trivially_relocatable.h>
+
 #include <cuda/std/utility>
```

Comment:
This is missing
```suggestion
#include <cuda/std/utility>
#include <cuda/std/type_traits>
```

or
```suggestion
#include <cuda/std/__type_traits/conjunction.h>
#include <cuda/std/utility>
```